vignettes/Million_KMs-Cycled-Per-Year.rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Estimating million km's cycled per zone in England and Wales regions"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing cycling potential estimates in UK cities}
```

Comment:
This should be updated to: Estimating million km's cycled per zone in England and Wales regions